cucumber/authenticators_jwt/features/authn_jwt_fetch_identity_decoded_token.feature

```diff
@@ -0,0 +1,191 @@
+    When I authenticate via authn-jwt with the JWT token
+    Then the HTTP response status code is 401
+    And The following appears in the log after my savepoint:
+    """
+    Errors::Conjur::RequiredSecretMissing: CONJ00037E Missing value for resource: cucumber:variable:conjur/authn-jwt/raw/token-app-property
+    """
+
+  Scenario: Host with slash as Token identity, identity-path configured, 200 ok
+    Given I have a "variable" resource called "test-variable"
+    And I successfully set authn-jwt "token-app-property" variable to value "host_claim"
+    And I extend the policy with:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !variable
+        id: identity-path
+    """
+    And I successfully set authn-jwt "identity-path" variable to value "some_policy/"
+    And I permit host "some_policy/sub_policy/host_test_from_token" to "execute" it
+    And I add the secret value "test-secret" to the resource "cucumber:variable:test-variable"
+    And I issue a JWT token:
+    """
+    {
+      "host_claim":"sub_policy/host_test_from_token",
+      "project-id": "myproject"
+    }
+    """
+    And I save my place in the log file
+    When I authenticate via authn-jwt with the JWT token
+    Then host "some_policy/sub_policy/host_test_from_token" has been authorized by Conjur
+    And I successfully GET "/secrets/cucumber/variable/test-variable" with authorized user
+    And The following appears in the log after my savepoint:
+    """
+    cucumber:host:some_policy/sub_policy/host_test_from_token successfully authenticated with authenticator authn-jwt service cucumber:webservice:conjur/authn-jwt/raw
+    """
+
+  Scenario: Host without slash as Token identity, identity-path configured, 200 ok
+    Given I have a "variable" resource called "test-variable"
+    And I successfully set authn-jwt "token-app-property" variable to value "host_claim"
+    And I extend the policy with:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !variable
+        id: identity-path
+    """
+    And I successfully set authn-jwt "identity-path" variable to value "some_policy/"
+    And I permit host "some_policy/host_test_from_token" to "execute" it
+    And I add the secret value "test-secret" to the resource "cucumber:variable:test-variable"
+    And I issue a JWT token:
+    """
+    {
+      "host_claim":"host_test_from_token",
+      "project-id": "myproject"
+    }
+    """
+    And I save my place in the log file
+    When I authenticate via authn-jwt with the JWT token
+    Then host "some_policy/host_test_from_token" has been authorized by Conjur
+    And I successfully GET "/secrets/cucumber/variable/test-variable" with authorized user
+    And The following appears in the log after my savepoint:
+    """
+    cucumber:host:some_policy/host_test_from_token successfully authenticated with authenticator authn-jwt service cucumber:webservice:conjur/authn-jwt/raw
```

Comment:
add log code

Reply:
This is success message not sure it has log code 

![image](https://user-images.githubusercontent.com/73157235/122670166-3b646900-d1c9-11eb-8261-9ce97d447c8a.png)